functional/ContentInputStream.java

```diff
@@ -171,7 +172,6 @@ public int read() throws IOException {
     }
 
     if (available <= 0) {
-      // Indicate EOF
```

Comment:
Is this comment inappropriate?

Reply:
that needs to be added back, it was a mistake to remove it.